duckdns/config.json

```diff
@@ -14,6 +14,8 @@
       "certfile": "fullchain.pem",
       "keyfile": "privkey.pem"
     },
+    "ipv4": "",
+    "ipv6": "",
```

Comment:
I don't think they should be as default config around

Reply:
I currently have that as the default because not setting the ip would make it autodetect which follows the same behavior.

Reply:
Sure, but you can also use jq for default value. I don't want show this settings default on UI. They can be around if user explicit set it.